### PR TITLE
Adds in search functionality to result-viewer, bugfixes for example

### PR DESCRIPTION
**Pull request by joelgriffith**
Fixes https://github.com/graphql/graphiql/issues/160

Also fixes a few issues found when using the `example` directory for testing. I could also write some scripts for watching fs changes to tighten the feedback loop when developing (currently you have to build and restart the example server).

### CODE REVIEW

File: css/codemirror.css [hunk 1]
```diff
@@ -333,3 +333,36 @@ div.CodeMirror-dragcursors {
 
 /* Help users use markselection to safely style text background */
 span.CodeMirror-selectedtext { background: none; }
```
Comment (joelgriffith) [inline css/codemirror.css]: Couldn't find a better spot to add these additions, which come straight from CodeMirror

File: example/build.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 rm -rf dist/ && mkdir -p dist/ &&
 babel server.js -o dist/server.js &&
```
Comment (joelgriffith) [inline example/build.sh]: According to README here, these _shoudln't_ pull from `node_modules`...

File: example/server.js [hunk 3]
```diff
@@ -23,20 +23,19 @@ import {
   GraphQLList,
 } from 'graphql';
 
-var app = express();
+const app = express();
 app.use(express.static(__dirname));
```
Comment (joelgriffith) [inline example/server.js]: This isn't the right path...

File: css/codemirror.css (hunk 1 above)
Comment (asiandrummer) [inline css/codemirror.css]: This is probably the best place to put these, so good choice.

File: example/build.sh (hunk 2 above)
Comment (asiandrummer) [inline example/build.sh]: If we make this change, we can probably remove the installation of `GraphiQL` from example/package.json

Comment (joelgriffith) [inline example/build.sh]: Hrm, I see that `graphiql` is referenced via `../graphiql`. I'm not sure why this wasn't working entirely locally... but I feel it's more obvious this way.

**Comment by asiandrummer** [comment]
@joelgriffith let me fix the lint errors really quick - you can rebase after that

**Comment by asiandrummer** [comment]
@joelgriffith I tried to attend to this lint issue, but it seems like on master there are none? Maybe we're running different lint configs?

**Comment by asiandrummer** [comment]
Never mind - I didn't install the correct version of eslint. Master should be clean now.

**Comment by joelgriffith** [comment]
Pulled in master, build passing 🎉

**Pull request merged**